datasets/loader.py

```diff
@@ -0,0 +1,386 @@
+# ===============================================================================
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import os
+import logging
+
+import pandas as pd
+import numpy as np
+
+from urllib.request import urlretrieve
+
+from sklearn.datasets import fetch_openml
+from sklearn.model_selection import train_test_split
+
+
+def a9a(dataset_dir=None):

```

Comment:
Why do you want to have the single file with dataset loaders? Why not to have the file for every dataset?

Reply:
To avoid a lot of imports at startup `load_datasets.py`. Now we get very small functions, I would leave them all in one file.